ReactiveSwift.playground/Pages/Property.xcplaygroundpage/Contents.swift

```diff
@@ -0,0 +1,290 @@
+ 
+ The `<~` operator can be used to bind properties in different ways. Note that in
+ all cases, the target has to be a binding target, represented by the [`BindingTargetProtocol`](https://github.com/ReactiveCocoa/ReactiveSwift/blob/master/Sources/UnidirectionalBinding.swift). All mutable property types, represented by the  [`MutablePropertyProtocol`](https://github.com/ReactiveCocoa/ReactiveSwift/blob/master/Sources/Property.swift#L28), are inherently binding targets.
+ 
+ * `property <~ signal` binds a [signal](#signals) to the property, updating the
+ property’s value to the latest value sent by the signal.
+ * `property <~ producer` starts the given [signal producer](#signal-producers),
+ and binds the property’s value to the latest value sent on the started signal.
+ * `property <~ otherProperty` binds one property to another, so that the destination
+ property’s value is updated whenever the source property is updated.
+ */
+scopedExample("Binding from SignalProducer") {
+    let producer = SignalProducer<Int, NoError> { observer, _ in
+        print("New subscription, starting operation")
+        observer.send(value: 1)
+        observer.send(value: 2)
+    }
+    let property = MutableProperty(0)
+    property.producer.startWithValues {
+        print("Property received \($0)")
+    }
+ 
+    // Notice how the producer will start the work as soon it is bound to the property
+    property <~ producer
+}
+
+scopedExample("Binding from Signal") {
+    let (signal, observer) = Signal<Int, NoError>.pipe()
+    let property = MutableProperty(0)
+    property.producer.startWithValues {
+        print("Property received \($0)")
+    }
+    
+    property <~ signal
+    
+    print("Sending new value on signal: 1")
+    observer.send(value: 1)
+    
+    print("Sending new value on signal: 2")
+    observer.send(value: 2)
+}
+
+scopedExample("Binding from other Property") {
+    let property = MutableProperty(0)
+    property.producer.startWithValues {
+        print("Property received \($0)")
+    }
+    
+    let otherProperty = MutableProperty(0)
+    
+    // Notice how property receives another value of 0 as soon as the binding is established
+    property <~ otherProperty
+    
+    print("Setting new value for otherProperty: 1")
+    otherProperty.value = 1
+
+    print("Setting new value for otherProperty: 2")
+    otherProperty.value = 2
+}
+/*:
+ ### Transformations
+ 
+ Properties provide a number of transformations like `map`, `combineLatest` or `zip` for manipulation similar to [signal](Signal) and [signal producer](SignalProducer)
+ */
+scopedExample("`map`") {
+    let property = MutableProperty(0)
+    let mapped = property.map { $0 * 2 }
+    mapped.producer.startWithValues {
+        print("Mapped property received \($0)")
+    }
+    
+    print("Setting new value for property: 1")
+    property.value = 1
+    
+    print("Setting new value for property: 2")
+    property.value = 2
+}
+
+scopedExample("`skipRepeats`") {
```

Comment:
Oh right, I created the more complex example for `uniqueValues` first and then decided to use the same sequence for `skipRepeats` as well to better illustrate the difference. 
I'll clean up the simple example.